components/select/select.ts

```diff
@@ -132,7 +132,51 @@ export class SelectComponent implements OnInit {
       this.hideOptions();
     }
   }
+  @Input()
+  public set selectIds(itemId:string){
```

Comment:
i think I  see what you mean, the setter method is empty. 
I added the logic of the setter inside the ngOnChanges method 
this is because it needs to run only when there is a change in the input (the current value is different from the previous) 
also, I was debugging and wanted to be checking the hooks to make sure the right thing is passed in the inputs at the right time.